exp/runtime/internal/controllers/extensionconfig_controller.go

```diff
@@ -149,11 +169,29 @@ func (r *Reconciler) reconcileDelete(_ context.Context, extensionConfig *runtime
 	return ctrl.Result{}, nil
 }
 
+func (r *Reconciler) secretToExtensionConfig(object client.Object) []reconcile.Request {
```

Comment:
nit: let's move this to the end of the file "out of the way" (as it's just a helper)

Reply:
Fair, I thought it is the right place because its the last func for the *Reconciler

Reply:
Makes sense. We don't have good conventions for that kind of stuff :)